control-plane/subcommand/server-acl-init/command_ent_test.go

```diff
@@ -1449,9 +1449,6 @@ func partitionedSetup(t *testing.T, bootToken string, partitionName string) *tes
 	server.Cfg.APIClientConfig.Token = bootToken
 	serverAPIClient, err := consul.NewClient(server.Cfg.APIClientConfig, 5*time.Second)
 	require.NoError(t, err)
-
 	_, _, err = serverAPIClient.Partitions().Create(context.Background(), &api.Partition{Name: partitionName}, &api.WriteOptions{})
-	require.NoError(t, err)
```

Comment:
is there a reason we have dropped this require check?

Reply:
Good catch! Didn't intend to to that. Restored in https://github.com/hashicorp/consul-k8s/pull/3435/commits/a31041315728b81eae2e00a0ed361edf514c1edd